livingdoc-confluence-plugin/src/main/java/info/novatec/testit/livingdoc/confluence/utils/RepositoryUtils.java

```diff
@@ -0,0 +1,18 @@
+package info.novatec.testit.livingdoc.confluence.utils;
+
+import info.novatec.testit.livingdoc.server.rpc.RpcServerService;
+
+public class RepositoryUtils {
+
+    private static final String BASE_TEST_URL_FORMAT = "%s/rpc/xmlrpc?handler=%s#%s";
```

Comment:
BASE_TEST_URL_FORMAT, REPOSITORY_BASE_URL_FORMAT maybe we don't need anymore

Reply:
We can set it as deprecated as soon as the next version will be released, in the current version 1.3.x is still make sense I guess

Reply:
OK, you are right